native/Cargo.toml

```diff
@@ -1,14 +1,14 @@
 [package]
 name = "okapi"
 version = "0.1.0"
-authors = ["Tomislav Markovski <tomislav@trinsic.id>"]
+authors = ["Tomislav Markovski <tomislav@trinsic.id>", "Scott Phillips <scott.phillips@trinsic.id>"]
 edition = "2018"
 # ensure this is not checked in as "true" as it will
 # cause vendored libraries to fail compilation due
 # to dependency on protoc compiler
 # if you need to compile .proto files, use this line:
-#build = "build.rs"
-build = false
+build = "build.rs"
+#build = false
```

Comment:
This needs be false and not commented out. The reason for that is that it makes NixOS builds fail.
A workaround would be to remove this file before packaging for nixos, but for now, let's just keep it to false when checking in.

Reply:
Yup, this was an oversight on my part. Thanks for catching this!